src/components/OntologyExplorer/index.tsx

```diff
@@ -58,6 +60,8 @@ export default function OntologyExplorer({ ontology, lattice, xref }: OntologyEx
     useState<DrawForceDagHighlightProps>(defaultForceHightlightProps);
   const [redrawCanvas, setRedrawCanvas] = useState<((p?: DrawForceDagHighlightProps) => void) | null>(null);
 
+  const windowHeight = useWindowHeight();
+
   const { dagCreateProps, cardWidth, cardHeight, menubarHeight, sugiyamaRenderThreshold } = state;
```

Comment:
Nit:  constants, such as menubarHeight, probably should not be in component state.